mage/docs/docs.go

```diff
@@ -74,7 +74,7 @@ func DocsPreview() {
 		time.Sleep(time.Second)
 	}
 
-	must.Run("open", "http://localhost:1313/docs/")
+	// must.Run("open", "http://localhost:1313/docs/")
```

Comment:
Is this tended change?

Reply:
Ah no, this wasn't intentional, commented this since the `open` command didn't work on my machine :/ Thanks for pointing this out!